cli/src/main.rs

```diff
@@ -52,6 +54,225 @@ pub fn main() {
         }
     };
 
+    let regtest = matches.is_present("regtest");
+    if regtest {
+        use std::ffi::OsString;
+        use std::fs::File;
+        use std::io;
+        use std::path::PathBuf;
+        use std::process::{Command, Stdio};
+        use std::{thread, time};
+
+        // confirm we are in a git directory, get info on it
+        let revparse_raw = Command::new("git")
+            .args(["rev-parse", "--show-toplevel"])
+            .output()
+            .expect("no git!? time to quit.");
+
+        let revparse = std::str::from_utf8(&revparse_raw.stdout).expect("revparse error");
+
+        let ident_raw = Command::new("git")
```

Comment:
Is this an accidentally left-over diagnostic?   What's the purpose of this variable (and `ident`)?  Maybe cut these 10 lines of code out?